dens-transactions/src/DensTransactions/DensTransactions.ts

```diff
@@ -1,3 +1,229 @@
-export function hello() {
-  console.log("Hello world");
-}
+import * as L from "lucid-cardano";
+import {
+  DensKey,
+  Protocol,
+  RecordDatum,
+  SetDatum,
+  SetInsert,
+} from "lbf-dens/LambdaBuffers/Dens.mjs";
+import { IsPlutusData } from "lbr-plutus/PlutusData.js";
+import * as Utils from "./Utils.js";
+// Scripts
+
+const initSetInsert = () => {
+  const setInsert: SetInsert = { setInsert: Utils.mkDensKey("") };
+  return Utils.toCslPlutusData(IsPlutusData[SetInsert].toData(setInsert));
+};
+
+export const mkProtocolOneShot = async (lucid: L.Lucid): Promise<L.OutRef> => {
+  const builder = new L.Tx(lucid);
+  const myAddr = await lucid.wallet.address();
+
+  const tx = builder.payToAddress(myAddr, { lovelace: BigInt(1) });
+
+  const complete = await tx.complete();
+  const signed = complete.sign();
+
+  const readyToSubmit = await signed.complete();
+  const hash = await readyToSubmit.submit();
+
+  await new Promise((r) => setTimeout(r, 10000));
```

Comment:
It seems we do a timeout here to wait for the transaction? Why not something like:
```suggestion
  await lucid.awaitTx(hash)
```